starknet_py/contract.py

```diff
@@ -1079,7 +1079,9 @@ def compute_address(
 
         compiled = create_compiled_contract(compiled_contract)
         assert compiled.abi is not None
-        translated_args = translate_constructor_args(compiled.abi, constructor_args)
+        translated_args = translate_constructor_args(
+            compiled.abi, constructor_args, cairo_version=0
```

Comment:
Could you please update the description of this function specifically to highlight that its usage is restricted to contracts written in Cairo 0? I will address compatibility for version 1 and higher in a separate PR.